src/devices/Mcp25xxx/samples/Mcp25xxx.Sample.cs

```diff
@@ -70,14 +70,13 @@ private static void ReadAllRegistersWithDetails(Mcp25xxx mcp25xxx)
             Console.WriteLine("Read All Registers");
             Console.WriteLine();
 
-
             ReadAllMessageTransmitRegistersWithDetails(mcp25xxx);
-            //ReadAllMessageReceiveRegistersWithDetails(mcp25xxx);
-            //ReadAllAcceptanceFilterRegistersWithDetails(mcp25xxx);
-            //ReadAllBitTimeConfigurationRegistersWithDetails(mcp25xxx);
-            //ReadAllErrorDetectionRegistersWithDetails(mcp25xxx);
-            //ReadAllInterruptRegistersWithDetails(mcp25xxx);
-            //ReadAllCanControlRegistersWithDetails(mcp25xxx);
+            ////ReadAllMessageReceiveRegistersWithDetails(mcp25xxx);
+            ////ReadAllAcceptanceFilterRegistersWithDetails(mcp25xxx);
+            ////ReadAllBitTimeConfigurationRegistersWithDetails(mcp25xxx);
+            ////ReadAllErrorDetectionRegistersWithDetails(mcp25xxx);
```

Comment:
mm not sure I understand why we got the extra double slash here

Reply:
Four slashes disable any other checks for these lines. Since it is commented out code (in an example), I think that's preferable over fixing the spacing. 
To comment out code (in stylecop-friendly way) you can either add four slashes or use two and a space. Of course, you wouldn't normally commit commented out code, except in samples.

Reply:
makes sense